server/src/instant/admin/routes.clj

```diff
@@ -263,6 +265,18 @@
   (let [{user-id :id app-id :app_id} (req->app-user! req)]
     (response/ok {:deleted (app-user-model/delete-by-id! {:id user-id :app-id app-id})})))
 
+(defn magic-code-post [req]
+  (let [email (ex/get-param! req [:body :email] email/coerce)
+        {user-id :id} (instant-user-model/get-or-create-by-email! {:email email})
```

Comment:
Afaik this is for the wrong model. This should be the `app-user-model` no?

Reply:
:o